flake.nix

```diff
@@ -23,6 +23,7 @@
       asus-pro-ws-x570-ace = import ./asus/pro-ws-x570-ace;
       asus-rog-strix-g513im = import ./asus/rog-strix/g513im;
       asus-rog-strix-g733qs = import ./asus/rog-strix/g733qs;
+      asus-zenbook-ux371 = import ./asus/zenbook/ux371;
```

Comment:
Ok. Now all that is missing is adding the model also to the markdown table in the README.md.

Reply:
Added